config.yaml

```diff
@@ -64,3 +64,70 @@ sdk:
           #
           # Environment variable: OTEL_TRACES_EXPORTER
           exporter: zipkin
+  # Configure the logger provider.
+  logger_provder:
+    # Log record exporters. Each exporter key refers to the type of the exporter. Values configure the exporter. Exporters must be associated with a log record processor.
+    exporters:
+      # Configure the otlp exporter.
+      otlp:
+        # Sets the protocol.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_PROTOCOL, OTEL_EXPORTER_OTLP_LOGS_PROTOCOL
+        protocol: http/protobuf
+        # Sets the endpoint.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_ENDPOINT, OTEL_EXPORTER_OTLP_LOGS_ENDPOINT
+        endpoint: http://localhost:4317/v1/logs
+        # Sets the certificate.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_CERTIFICATE, OTEL_EXPORTER_OTLP_LOGS_CERTIFICATE
+        certificate: /app/cert.pem
+        # Sets the mTLS private client key.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_CLIENT_KEY, OTEL_EXPORTER_OTLP_LOGS_CLIENT_KEY
+        client_key: /app/cert.pem
+        # Sets the mTLS client certificate.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE, OTEL_EXPORTER_OTLP_LOGS_CLIENT_CERTIFICATE
+        client_certificate: /app/cert.pem
+        # Sets the headers.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_HEADERS, OTEL_EXPORTER_OTLP_LOGS_HEADERS
+        headers:
+          api-key: 1234
+        # Sets the compression.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_COMPRESSION, OTEL_EXPORTER_OTLP_LOGS_COMPRESSION
+        compression: gzip
+        # Sets the max time to wait for each export.
+        #
+        # Environment variable: OTEL_EXPORTER_OTLP_TIMEOUT, OTEL_EXPORTER_OTLP_LOGS_TIMEOUT
+        timeout: 10000
+    # List of log record processors. Each log record processor has a name and args used to configure it.
+    log_record_processors:
```

Comment:
I know the "appender"/handler is language-specific, but I think it is essential to the logging SDK configuration. Should this minimal config define some mechanism for it?

Reply:
Good question.

The wiring up of appenders to the SDK seems more like an instrumentation concern than an SDK concern. I suggest we include that under the `instrumentation` section. I.e. something like:
```
sdk:
  ...
instrumentation:
  log_appenders:
    - name: log4j
    - name: logback
```

Reply:
This looks good.

Reply:
Created a separate issue for configuring handlers, feel free to resolve that issue in this PR or separately.